documentation/assemblies/upgrading/assembly-upgrade-kafka.adoc

```diff
@@ -5,16 +5,54 @@
 [id='assembly-upgrade-kafka-{context}']
 = Strimzi and Kafka upgrades
 
-Upgrading Strimzi is a two-stage process. To upgrade brokers and clients without downtime, you _must_ complete the upgrade procedures in the following order:
+Upgrading Strimzi is a three-stage process. To upgrade brokers and clients without downtime, you _must_ complete the upgrade procedures in the following order:
 
 . Update your Cluster Operator to the latest Strimzi version.
-** xref:assembly-upgrade-cluster-operator-{context}[]
++
+The approach you take depends on how you xref:cluster-operator-{context}[deployed the Cluster Operator].
++
+* If you deployed the Cluster Operator using the installation YAML files, perform your upgrade by modifying the Operator installation files, as described in xref:assembly-upgrade-cluster-operator-{context}[Upgrading the Cluster Operator]. 
++
+* If you deployed the Cluster Operator from {OperatorHub}, use the Operator Lifecycle Manager (OLM) to change the update channel for the Strimzi Operators to the new Strimzi version.
++
+Depending on your chosen upgrade strategy, after updating the channel, either:
++
+** An automatic upgrade is initiated
++
+** A manual upgrade will require approval before the installation begins
++
+For more information on using {OperatorHub} to upgrade Operators, see the {OLMOperatorDocs}.
 
 . Upgrade all Kafka brokers and client applications to the latest Kafka version.
-** xref:assembly-upgrading-kafka-versions-{context}[]
++
+* xref:assembly-upgrading-kafka-versions-{context}[]
+* xref:con-strategies-for-upgrading-clients-{context}[] 
+
+. If applicable, update existing resources to handle deprecated custom resource properties.
++
+* xref:assembly-upgrade-resources-{context}[Strimzi resource upgrades]
+
+.Additional upgrade tasks
+
+You might also need to complete the following upgrades.
+
+The old listener configuration is deprecated. 
+We recommend that you xref:con-upgrade-listeners-{context}[update listeners to use the `GenericKafkaListener` schema], where `listeners` is configurable as an array.
+
+The incremental cooperative rebalance protocol was introduced in Kafka 2.4.0. 
```

Comment:
Do we need to say this here?

Reply:
I'd like to see these options as bullet points

Reply:
Done. I changed to bullet points.